src/main/scala/io/qbeast/spark/delta/writer/BlockWriter.scala

```diff
@@ -131,6 +133,8 @@ case class BlockWriter(
         .setModificationTime(fileStatus.getModificationTime())
         .setRevisionId(revision.revisionID)
         .result()
+
+      logInfo(s"Qbeast: Adding file $file")
```

Comment:
Do we want to print the whole file information?

Reply:
For info level, I'd say that adding the whole file information is too much, so I'd suggest one of two options:

1- Convert this message to debug level and add the whole file information.
2- Add another message at debug level that prints the whole file information.


I'm comfortable with both options, what do you think?

Reply:
I would go for the second option: keep the whole file information for debug.
Possibly for info we can print only the path.